HW03/aleksei_zhuchkov/spec/hw_03_t_03_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative '../aleksei_zhuchkov_hw_03_t_03'
+
+RSpec.describe 'Homework3' do
+  let(:obj_hw3) { Homework3.new }
+
+  context 'when testing basic things' do
+    it 'return value is string type' do
+      expect(obj_hw3.task3('')).to be_an_instance_of(String)
+    end
+
+    it 'return value "0" (string) when get empty string' do
```

Comment:
do not mix what should be written in it block and in the context

Reply:
I have done it
https://github.com/labs-ruby/homeworks-2022/pull/122/commits/cddd4094544ca3defa0eb50d399968fc364b1649